test/integration/targets/rabbitmq_lookup/tasks/tests.yml

```diff
@@ -0,0 +1,73 @@
+- name: Install pika
+  pip:
+    name: pika
+    state: latest
+
+# Rabbitmq lookup
+- set_fact:
+    test_rabbitmq: True
+
+- name: Test that giving an incorrect amqp protocol in URL will error
+  set_fact:
+    rabbitmq_test_protocol: "{{ lookup('rabbitmq', url='zzzamqp://guest:guest@192.168.250.1:5672/%2F', channel='hello', count=3) }}"
+  ignore_errors: yes
+  register: rabbitmq_protocol_error
+
+# If pika is not installed, don't carry out rest of tests.
```

Comment:
How could pika not be installed here ?

Reply:
Some sort of mocks I guess.  I have a rabbitmq message publish module PR open.. when that goes through, this testing will be extended to actually publish a message to the queue (via the module) and this lookup plugin will receive it.

Reply:
I don't understand: `pika` Python package is installed in a task just above in `Install pika` task.

Reply:
Okay, I misunderstood. Sorry.   I have removed that comment now.  Thanks!